examples/ltc-example/src/app/ltc.virtualAccount.example.ts

```diff
@@ -0,0 +1,76 @@
+import { TatumLtcSDK } from '@tatumio/ltc'
+
+export async function ltcVirtualAccountExample() {
+  const ltcSDK = TatumLtcSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+  // Create an extended public key
+  const { xpub } = await ltcSDK.wallet.generateWallet(undefined, { testnet: true })
```

Comment:
Do we need to set here testnet? It is not testnet/mainnet choosen by selection of API key?

Reply:
It's not an API method, by default it generates mainnet and I hade invalid network version errors if not supplied